src/main/java/org/codehaus/mojo/taglist/TaglistReportRenderer.java

```diff
@@ -199,12 +162,8 @@ private void doTagDetailedPart(TagReport tagReport) {
             return;
         }
 
-        sink.section2();
-        sink.sectionTitle2();
-        sink.anchor(tagReport.getHTMLSafeLinkName());
-        sink.text(tagReport.getTagName());
-        sink.anchor_();
-        sink.sectionTitle2_();
+        startSection(tagReport.getTagName(), tagReport.getHTMLSafeLinkName());
+
         sink.paragraph();
         sink.bold();
```

Comment:
This one use sink direct - there are no methods like start/stop paragraph and also also for bold text.

Alternatively we can use paragraph without bold

Reply:
Keep the bold, if you need bold, no problem.

Reply:
bold is not required .. code looks easier